Package.resolved

```diff
@@ -56,30 +56,21 @@
         }
       },
       {
-        "package": "Unbox",
-        "repositoryURL": "https://github.com/JohnSundell/Unbox.git",
-        "state": {
-          "branch": null,
-          "revision": "17ad6aa1cc2f1efa27a0bbbdb66f79796708aa95",
-          "version": "2.5.0"
-        }
-      },
-      {
         "package": "Yams",
         "repositoryURL": "https://github.com/jpsim/Yams.git",
         "state": {
           "branch": null,
-          "revision": "5ac9248b378b5e12201c255ba74caf990d1e0f4b",
-          "version": "0.3.6"
+          "revision": "05921531200acff0d8d9caab5ec77b2bcf300b6b",
+          "version": "0.3.7"
```

Comment:
I couldn't stop this from updating (still a little new to the SwiftPM).  After looking at the Yams repo, it doesn't appear much changed from `0.3.6` to `0.3.7` that would impact XcodeGen.

Reply:
That's fine